hladnymatfyzak/__init__.py

```diff
@@ -151,16 +151,17 @@ def ffood(which, day=None, month=None, year=None):
     if month is None:
         month = int(time.strftime("%m"))
     weekday = int((datetime.date(year, month, day)).weekday())
-
     req = requests.get(FF_URL)
     soup = BeautifulSoup(req.text)
     menu_week = soup.find_all('ul')
 
     menu_week = menu_week[index]
     menu_bs = BeautifulSoup(str(menu_week))
+    # one ul for every day excepts the first for navigation menu
+    weekday = weekday + 1
     daymenu = menu_bs.find_all('ul')[weekday]
 
-    # weekday should be <1,5>
+    # weekday should be <0,4>
```

Comment:
@tWido is this comment in the right place? I believe it would be way more useful somewhere before line 161.